plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/FacetUtil.java

```diff
@@ -120,6 +131,7 @@ public FacetUtil addWebFacetToBatch(IProjectFacetVersion webFacet) {
       return this;
     }
 
+    // FIXME: should this try to find web.xml first?
```

Comment:
Cursory examination, I didn't really understand the intention of this code.  Will add a comment.